packages/react/src/hooks/cloud/krisp/useKrispNoiseFilter.ts

```diff
@@ -6,33 +6,38 @@ import type { TrackReferenceOrPlaceholder } from '@livekit/components-core';
 import { useLocalParticipant } from '../../..';
 
 /**
- * @alpha
+ * @beta
  */
 export interface useKrispNoiseFilterOptions {
   /**
-   * by default the hook will use the localParticipant's microphone track publication.
-   * You can override this behavior by passing in a target TrackReference here
+   * The track reference to use for the noise filter (defaults: local microphone track)
    */
   trackRef?: TrackReferenceOrPlaceholder;
+  /**
+   * @internal
```

Comment:
why would they be internal?

Reply:
this was the easiest way to just leave it out of the docs. this comes from the closed-source underlying krisp SDK, so there's no good way to document this parameter. the actual values available to tweak here also seem unlikely to be useful, it's just debugging flags basically? I don't feel strongly about it I just thought it's easier to exclude it from the docs.